TrailBlaze/Networking/PaceImprovementTracker.m

```diff
@@ -26,7 +26,10 @@ + (BOOL) isAtStartPosition: (CLLocation *) userLocation firstPoint: (NSArray *)
 
 - (instancetype)initWithRunID: (NSString *) objectId {
     self.pedometer = [[CMPedometer alloc] init];
-    self.currentPacesArray = [NSMutableDictionary new];
+    self.currentPacesDictionary = [NSMutableDictionary new];
+    self.i = 1;
```

Comment:
curious, why does this value need to start at 1?

also would recommend giving this a better descriptive name

Reply:
i, now renamed to indexOfNextPoint, starts at 1 because at the start of the run it is assumed that the runner has already passed the first point of the route polyline so the next point to pass would be the point at index 1 of the polylinePoints array